File: 2310817120009_AlysaArmelia_PemrogramanI_Modul5/Python/PRAK504-2310817120009-AlysaArmelia.py
def reverse(hitungan):

    reversed_beberapa = 0

    while hitungan != 0:
        pengingat = hitungan % 10
        reversed_beberapa = reversed_beberapa * 10 + pengingat
        hitungan //= 10

    return reversed_beberapa

A, B = map(int, input().split())
A = reverse(A)
B = reverse(B)
C = A + B
print(reverse(C))

File: 2310817120009_AlysaArmelia_PemrogramanI_Modul5/Bahasa C/PRAK504-2310817120009-AlysaArmelia.c
#include <stdio.h>

int reverse (int hitungan){ 

    int reversed_beberapa = 0;

    while (hitungan != 0)
    {
        int pengingat = hitungan % 10;
        reversed_beberapa = reversed_beberapa * 10 + pengingat;
        hitungan /= 10;
    }
    
    return reversed_beberapa;
 } 

int main() { 
    int A, B; 
    scanf("%d %d",&A,&B); 
    A = reverse(A); 
    B = reverse(B); 
    int C = A+B; 
    printf("%d", reverse(C)); 
}